novuhq/website
# feat: add the detail Contributor page

**Describe what changes this pull request brings**
This pull request adds the detail `Contributor` page.


**Steps to test**
1. Open [the page]()
2. Confirm that everything looks and works as expected
